SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Supported Versions
+
+The [`master` branch](https://github.com/reactos/reactos/tree/master) and [latest release](https://github.com/reactos/reactos/releases) are both supported with security updates.
```

Comment:
I do try to have my eyes on security vulnerabilities as well, yes.
But since the current development status of ReactOS is so much Alpha, my main motivation for the backportds is currently is weighting more what is relatively low-risk, and what has a good ratio of "bang--per-size". My goal with the current rls-branches 0.4.7 to 0.4.14 is definitely not yet generating a safe OS that would be usable for desktop usage. We are far away from that still.
Basically each crash that ros currently runs into can also be perceived as a security-vulnerability, and we do still have tons of them!
And I definitely do not want to be obliged yet to fix back all security fixes. That is not yet worth the risk.
We will get there at some point, but not in the coming years. Sorry, I don't want to promise anything like that yet.
Stating anything like that within some official document would be a blatant lie.
We do have to face the truth: ReactOS currently **is** very vulnerable, and it will stay like that for the coming years.